### PR TITLE
Defined a Post Model in models.py

### PR DESCRIPTION
**Pull request by brianwmlo**
Created Post model with title, content, created, and last_updated

### CODE REVIEW

File: blogger/post/models.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
 from django.db import models
+from django.utils import timezone
```
Comment (derekkwok) [inline blogger/post/models.py]: This import statement is not needed.

File: blogger/post/models.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
 
 # Create your models here.
+class Post(models.Model):
+	title = models.CharField(max_length=200)
+	content = models.Textfield()
+	created = models.DateTimeField(auto_now_add=True)
```
Comment (derekkwok) [inline blogger/post/models.py]: Let's rename "created" to "created_on". Thoughts?

File: blogger/post/models.py [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+	last_updated = models.DateTimeField(auto_now=True)
```
Comment (derekkwok) [inline blogger/post/models.py]: For consistency purposes, what if we named this "updated_on"?

File: blogger/post/models.py (hunk 1 above)
Comment (brianwmlo) [reply blogger/post/models.py]: ok noted thanks

File: blogger/post/models.py (hunk 2 above)
Comment (brianwmlo) [reply blogger/post/models.py]: sounds good will change

File: blogger/post/models.py (hunk 3 above)
Comment (brianwmlo) [reply blogger/post/models.py]: yup lets do that

**Pull request merged**